media-gfx/krita/krita-5.2.2.ebuild

```diff
@@ -97,9 +97,9 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}/${PN}-4.3.1-tests-optional.patch"
 	"${FILESDIR}/${PN}-5.1.5-sip-6.8.patch" # bug 919139
-	"${FILESDIR}/${PN}-5.1.5-libjxl-0.9.patch" # bug 922524
-	"${FILESDIR}/${PN}-5.2.2-fftw.patch"
-	"${FILESDIR}/${PN}-5.2.2-openjpeg.patch"
+	"${FILESDIR}/${P}-libjxl-0.9.patch" # bug 922524
```

Comment:
Why did you remove the version information here?
Usually these are used to indicated which version the patches were introduced or started to be relevant.

Reply:
File names are still the same, I've just shortened their representations a bit for this particular ebuild.
I believe it's a matter of taste, but [the documentation](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/) recommends using `${P}`.

> Patches are best named ${P}-what-it-does.patch

> When mirroring patches, choosing a name that will not cause conflicts is important — the ${P} prefix is highly recommended here.

Also using `${P}` forces a developer who wants to perform a version bump to check if the patch is still relevant, so they should decide whether patch might be dropped or the corresponding ebuild line should be adjusted.

And it's widely used, although `${PN}` is more common.
```shell
# pwd
/var/db/repos/DarkDefender
# ag '{P}.*.patch' | wc -l
6323
# ag '{PN}-[0-9].*.patch' | wc -l
8281
```

But of course, I can amend my commit to use `${PN}` instead of `${P}`.
Do you want me to do so?